AWSCore/Utility/AWSCategory.m

```diff
@@ -500,6 +500,11 @@ - (AWSRegionType)aws_regionTypeValue {
         || [self isEqualToString:@"eu-central-2"]) {
         return AWSRegionEUCentral2;
     }
+    if ([self isEqualToString:@"AWSRegionILCentral1"]
+        || [self isEqualToString:@"ILCentral1"]
+        || [self isEqualToString:@"il-central-1"]) {
+        return AWSRegionILCentral1;
+    }
```

Comment:
is there an order to this? if not, let's add it to the end of the if statements?